Docs/source/conf.py

```diff
@@ -58,7 +61,18 @@
 
 # Relative path to bibliography file, bibliography style
 bibtex_bibfiles = ['./refs.bib']
-bibtex_default_style = 'unsrt'
+
+class WarpXBibStyle(UnsrtStyle):
```

Comment:
Cool! Can you please add inline comments or class doc strings to the reference for these options?

Reply:
Yes, no problem. I'd add some comments and a link to the Sphinx documentation.